front/app/modules/commercial/smart_groups/components/UserFilterConditions/PredicateSelector.tsx

```diff
@@ -16,6 +16,78 @@ type Props = {
   onChange: (predicate: TPredicate) => void;
 };
 
+const PREDICATE_MESSAGES: Record<TPredicate, MessageDescriptor> = {
```

Comment:
This is not directly related to removing participants- just an improvement for more type safety.